grails-app/controllers/au/org/ala/phyloviz/CharactersController.groovy

```diff
@@ -12,17 +13,12 @@ class CharactersController {
      *
      * @return
      */
-    def list(){
-        def id = authService.getUserId()
-        log.debug(id)
-        def owner = Owner.findByUserId(id)
-        def lists = Characters.findAllByOwner(owner);
-        def slist = Characters.findAllByOwner(Owner.findByUserId(1));
-        lists.addAll(slist);
-        log.debug(lists)
-        def result = charactersService.getCharUrl(lists);
+    def list() {
+        List characterLists = charactersService.getCharacterListsByOwner()
 
-        if(params.callback){
+        List result = treeService.filterCharacterListsByTree(params.treeId, characterLists)
```

Comment:
This isn't necessary because groovy will automatically convert it. It doesn't hurt though.